spec/controllers/api/v1/cakes_controller_spec.rb

```diff
@@ -93,6 +93,11 @@
       it "deletes the specified cake" do
         expect(Creation.exists?(cake.id)).to be_falsey
       end
+
+      it 'returns an empty json response' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.